alws/crud/build.py

```diff
@@ -88,6 +95,36 @@ async def create_build(
     await db.commit()
     await db.refresh(db_build)
     start_build.send(db_build.id, build.model_dump())
+    if settings.github_integration_enabled:
+        try:
+            github_client = await get_github_client()
+            repos = set()
+            for task in build.tasks:
+                if isinstance(task, build_schema.BuildTaskModuleRef):
+                    repos.add(f"module {task.module_name}")
+                    continue
+
+                repos.add(f"{task.url} {task.git_ref}")
+            issues = await find_issues_by_repo_name(
+                github_client=github_client,
+                repo_names=list(repos),
+            )
+            if issues:
+                await set_build_id_to_issues(
+                    github_client=github_client,
+                    issues=issues,
+                    build_id=db_build.id,
+                )
+                await move_issues(
+                    github_client=github_client,
+                    issues=issues,
+                    status="Building",
```

Comment:
It would be great to turn these statuses into an enum in albs-github-integration, what do you think? It's not urgent, we can create a ticket for that one and, since it's an easy one, anybody (even new contributors) can take it in the future.